src/dmd/backend/dwarfdbginf.d

```diff
@@ -45,15 +45,17 @@ import dmd.backend.cdef;
 
 version(Windows)
 {
+    nothrow
     extern (C) char* getcwd(char* buffer, size_t maxlen);
+    nothrow
     extern (C) int* _errno();   // not the multi-threaded version
 }
 else
 {
     import core.sys.posix.unistd : getcwd;
 }
 
-static if (ELFOBJ || MACHOBJ)
+static if (1)
```

Comment:
Dont' think you need to keep this around, unless you really insist on adjusting indents later.  It doesn't make reviewing an issue due to `w=1` or `git -w` being able to cope with ignoring whitespace.

Reply:
> It doesn't make reviewing an issue due to w=1 or git -w being able to cope with ignoring whitespace.

I still don't care to mix that up with heavy changes.